### PR TITLE
[12.0][MIG] sale_stock_delivery_address

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
**sale_stock_delivery_addres** migration to v12

cc ~ @LoisRForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:  Works as expected

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: sale_stock_delivery_address/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Sale Stock Sourcing Address",
```
Comment (MiquelRForgeFlow) [inline sale_stock_delivery_address/__manifest__.py]:
```suggestion
    "name": "Sale Stock Delivery Address",
```
maybe

**Comment by LoisRForgeFlow** [comment]
@rousseldenis @JordiBForgeFlow can any of you merge this one?

**Comment by LoisRForgeFlow** [comment]
Thanks!

**Pull request merged**